src/torrent.cpp

```diff
@@ -5707,6 +5714,8 @@ namespace {
 
 		std::vector<boost::uint8_t> const new_priority = fix_priorities(files, m_torrent_file->files());
 
+		if (m_file_priority == new_priority) return;
```

Comment:
see my comment on this in my code review

Reply:
You're right, let's drop those checks for now.